.github/workflows/tests.yaml

```diff
@@ -1,16 +1,14 @@
 name: tests
 
 on:
+  workflow_dispatch:
   pull_request:
     paths-ignore:
       - 'CHANGELOG.md'
       - 'README.md'
       - 'MAINTAINERS'
 
   push:
-    branches:
-      - 'main'
-      - 'release/**'
```

Comment:
I don't mind the `workflow_dispatch` addition but please undo the branch deletions

Reply:
Haha yeah just added temporarily so I could run the tests on my fork